python/src/GaussianProcess_doc.i.in

```diff
@@ -82,6 +82,29 @@ isTrendStationary : bool
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::GaussianProcess::getSamplingMethod
+"Get the used method for getRealization.
+
+Returns
+-------
+samplingMethod : int
+    Used method for sampling.
+
+Notes
+-----
+
+Available parameters are :
+
+  * 0 : Cholesky factor sampling (default method)
+
+  * 1 : H-Matrix method (if H-Mat available)
+
+  * 2 : Gibbs method (in dimension 1 only)
```

Comment:
Maybe mention the "names" of the possibilities - not necessarily here but at least in the `setSamplingMethod` doc. Something like:
```suggestion
  * 0 (CHOLESKY) : Cholesky factor sampling (default method)

  * 1 (HMAT) : H-Matrix method (if H-Mat available)

  * 2 (GIBBS) : Gibbs method (in dimension 1 only)
```
It is probably better if we promote the use of these names rather than directly the integers since users benefit from autocompletion and this is less error-prone.